pkg/distro/distro.go

```diff
@@ -41,6 +41,9 @@ type Distro interface {
 	// Returns the name of the distro.
 	Name() string
 
+	// Returns the nick of the distro.
```

Comment:
What is the nick? A nickname?

Reply:
I know we had this already but I think a more correct term for it is "codename".
> Codename: Ootpa (is based on Fedora 28, upstream Linux kernel 4.18, systemd 239, and GNOME 3.28.)
https://access.redhat.com/articles/3078

> VERSION_CODENAME=
> A lower-case string (no spaces or other characters outside of 0–9, a–z, ".", "_" and "-") identifying the operating system release code name
https://www.freedesktop.org/software/systemd/man/latest/os-release.html

Let's rename it to `codename`.  I think it'll be clearer.

Reply:
I've submitted my comment without refreshing the page. Will rename it :)

Reply:
I renamed `Nick` -> `Codename`